### ISSUE TITLE
Grammar edit. 

### ISSUE THREAD

**Issue by benreckas** [benreckas/the-dev-toolkit#7]
Improve some descriptions. General grammar improvements. Moving forward- grammar and formatting will be required to submit a contribution.

**Comment by maciekChmura** [comment]
Hi, is this issue still relevant?

**Comment by benreckas** [comment]
@maciekChmura Absolutely, I've been wanting to refresh this resource a bit and add some stuff to it. Feel free to make any PR's you think are relevant. 🙂
